.circleci/config.yml

```diff
@@ -19,7 +19,7 @@ commands:
 executors:
   python:
     docker:
-      - image: cimg/python:3.9.10
+      - image: cimg/python:3.9.12
```

Comment:
question: did you mean to update the Python version? If so, can we update the dockerfile too?
https://github.com/mozilla/fx-private-relay/blob/41a4d562504713593677b4564101fbd2727e9f67/Dockerfile#L14

Reply:
yes (I meant to update Python) and yes (I will update the Dockerfile)!